From 22b6572bf5a9b6de6b1c2315655aab71d8071f8d Mon Sep 17 00:00:00 2001
From: henr1kas <161259379+henr1kas@users.noreply.github.com>
Date: Tue, 19 May 2026 11:43:05 +0200
Subject: [PATCH] Update decryption keys

---
 samloader/auth.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/samloader/auth.py b/samloader/auth.py
index fc4d482..71177b0 100644
--- a/samloader/auth.py
+++ b/samloader/auth.py
@@ -7,8 +7,8 @@ import base64
 from Cryptodome.Cipher import AES

 # Constant key input values.
-KEY_1 = "vicopx7dqu06emacgpnpy8j8zwhduwlh"
-KEY_2 = "9u7qab84rpc16gvk"
+KEY_1 = "jqw2zntssasudk3zuwwx4tnqz5h3b6i4"
+KEY_2 = "ifzg4ovyl3cyl3a5"

 # PKCS#7 padding functions.
 pkcs_unpad = lambda d: d[:-d[-1]]
-- 
2.54.0
