From 3d7e45b46175118f6b70e41c35d3569030da0557 Mon Sep 17 00:00:00 2001
From: Salvo Giangreco <giangrecosalvo9@gmail.com>
Date: Thu, 21 Aug 2025 22:01:23 +0200
Subject: [PATCH] Add timeout to version.xml request

---
 samloader/versionfetch.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samloader/versionfetch.py b/samloader/versionfetch.py
index 830f917..6a65701 100644
--- a/samloader/versionfetch.py
+++ b/samloader/versionfetch.py
@@ -18,7 +18,7 @@ def normalizevercode(vercode: str) -> str:
 def getlatestver(model: str, region: str) -> str:
     """ Get the latest firmware version code for a model and region. """
     req = requests.get("https://fota-cloud-dn.ospserver.net/firmware/" \
-        + region + "/" + model + "/version.xml" , headers={"User-Agent": "Kies2.0_FUS"})
+        + region + "/" + model + "/version.xml" , headers={"User-Agent": "Kies2.0_FUS"}, timeout=3)
     if req.status_code == 403:
         print("Error 403 : Model or region not found (403) | Or TAC is not for " + model)
         exit()
-- 
2.50.1

